cmd/daemon/main.go

```diff
@@ -259,7 +260,7 @@ func main() {
 	dnsSetter := dns.NewSetter(infoSubject)
 	dnsHostSetter := dns.NewHostsFileSetter(dns.HostsFilePath)
 
-	eventsDbPath := fmt.Sprintf("%smoose.db", internal.DatFilesPath)
+	eventsDbPath := path.Clean(internal.DatFilesPath + "/moose.db")
```

Comment:
using `filepath.Join` might be cleaner, and don't need to worry about forgetting `/`.